src/govuk/components/all.template.test.js

```diff
@@ -0,0 +1,148 @@
+const { HtmlValidate } = require('html-validate')
+require('html-validate/jest')
+
+// We can't use the render function from jest-helpers, because we need control
+// over the nunjucks environment.
+const nunjucks = require('nunjucks')
+
+const { allComponents, getComponentData } = require('../../../lib/file-helper')
+const { nunjucksEnv, renderSass, renderHtml } = require('../../../lib/jest-helpers')
+
+const configPaths = require('../../../config/paths.js')
+
+describe('Components', () => {
```

Comment:
Separated Nunjucks and HTML tests from [src/govuk/components/all.test.js](https://github.com/alphagov/govuk-frontend/pull/2878/files#diff-e5806587e66dd93c9ab15b1591df5f8fbc6c631ec228c68518d68f93ad69e50c) to avoid loading the `@jest-environment puppeteer` (wasted a perfectly good Chromium window)